templates/terraform/constants/resource_dns_resource_record_set.go.erb

```diff
@@ -0,0 +1,13 @@
+func ipv6AddressforDnsDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+    if d.Get("type") == "AAAA" {
+        return AddressDiffSuppress(k, old, new, d)
+    }
+    return false
+}
+
+func AddressDiffSuppress(_, old, new string, _ *schema.ResourceData) bool {
```

Comment:
Can these two functions be combined? Or is there some advantage for separating?

Reply:
I can't run the unit tests if i combine them.